lib/broccoli/ember-app.js

```diff
@@ -38,6 +38,8 @@ var merge         = require('lodash/object/merge');
 var omit          = require('lodash/object/omit');
 var Funnel        = require('broccoli-funnel');
 
+var deprecate     = require('../utilities/deprecate');
```

Comment:
@rwjblue So far as I can tell there are currently no deprecations inside of lib/broccoli, and the `ui` object from the task isn't available.

Would you like me to pass it through here? https://github.com/ember-cli/ember-cli/blob/master/lib/models/builder.js#L55